content/sensu-go/5.16/api/roles.md

```diff
@@ -27,10 +27,29 @@ The `/roles` API endpoint provides HTTP GET access to [role][1] data.
 The following example demonstrates a request to the `/roles` API endpoint, resulting in a JSON array that contains [role definitions][1].
 
 {{< highlight shell >}}
-curl http://127.0.0.1:8080/api/core/v2/namespaces/default/roles -H "Authorization: Bearer $SENSU_TOKEN"
+curl -X GET http://127.0.0.1:8080/api/core/v2/namespaces/default/roles
```

Comment:
requires ' \ ' at the end of the line, probably should move 'curl -X GET' to it's own line for consistency